### PR TITLE
fontlint: don't throw an error when no more args are expected

### PR DESCRIPTION
**Pull request by austin987**
Provide a general summary of your changes in the **Title** above.

### Motivation and Context
- [X] Why is this change required? What problem does it solve?
Fixes: /usr/bin/fontlint line: 256 Integer expression out of bounds in array

- [ ] If it fixes an open issue, include the text `Closes #1` (where 1 would be the issue number) to your commit message.

### Types of changes
What types of changes does your code introduce? Check all the boxes that apply:
- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Description
- [X] Describe your changes in detail.
Noticed while testing some fonts on my system.

### Final checklist
Go over all the following points and check all the boxes that apply. 
If you're unsure about any of these, don't hesitate to ask. We're here to help! Various areas of the codebase have been worked on by different people in recent years, so if you are unfamiliar with the general area you're working in, please feel free to chat with people who have experience in that area. See the list [here](https://github.com/fontforge/fontforge/blob/master/CONTRIBUTING.md#people-to-ask).
- [X] My code follows the code style of this project found [here](https://github.com/fontforge/fontforge/blob/master/CONTRIBUTING.md#coding-style).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the [**CONTRIBUTING**](https://github.com/fontforge/fontforge/blob/master/CONTRIBUTING.md) guidelines.

Your pull request will be tested via Travis CI to automatically indicate that your changes do not prevent compilation. FontForge is a big program, so Travis can easily take over 20 minutes to confirm your changes are buildable. Please be patient. More details about using Travis can be found [here](https://github.com/fontforge/fontforge/blob/master/CONTRIBUTING.md#using-travis-ci).  
  
If it reports back that there are problems, you can log into the Travis system and check the log report for your pull request to see what the problem was. If no error is shown, just re-run the Travis test for your pull-request (that failed) to see a fresh report since the last report may be for someone else that did a later pull request, or for mainline code. If you add new code to fix your issue/problem, then take note that you need to check the next pull request in the Travis system. Travis issue numbers are different from GitHub issue numbers.

### CODE REVIEW

File: fontforgeexe/fontlint.pe [hunk 1]
```diff
@@ -250,9 +250,11 @@ while ($argc>1)
       endloop
       if (i<Strlen($argv[1]))
         optarg=Strsub($argv[1],i);
-      else
+      elif ($argv[2])
```
Comment (jtanx) [inline fontforgeexe/fontlint.pe]: elif -> elseif?

Comment (frank-trampe) [inline fontforgeexe/fontlint.pe]: Pinging @austin987.

**Comment by austin987** [comment]
Sorry for delay, been busy part couple weeks. I will did and retest this week.

Thanks for your attention to this issue.

**Comment by austin987** [comment]
@jtanx / @fabiangreffrath, updated, and checks are passing.

Sorry again for my delay!

**Comment by jtanx** [comment]
@austin987 can you please provide an example, e.g. `fontlint some-font.ttf`? Just going off what arguments fontlint expects, I'm not entirely sure what this is fixing.

**Comment by jtanx** [comment]
Ping

**Comment by austin987** [comment]
I can't find the font I originally used to replicate this; sorry for the waste of time.

**Pull request closed without merging**